src/components/Popups/PopupReply.tsx

```diff
@@ -2,32 +2,34 @@ import React from 'react';
 import Button from 'box-ui-elements/es/components/button';
 import PrimaryButton from 'box-ui-elements/es/components/primary-button';
 import { KEYS } from 'box-ui-elements/es/constants';
-import { FormattedMessage } from 'react-intl';
+import { Editor, EditorState } from 'draft-js';
+import { FormattedMessage, IntlShape } from 'react-intl';
 import messages from './messages';
 import PopupBase from './PopupBase';
 import ReplyField from './ReplyField';
 import './PopupReply.scss';
 
 export type Props = {
     className?: string;
-    defaultValue?: string;
+    editorState: EditorState;
+    intl: IntlShape;
     isPending: boolean;
-    onCancel: (text?: string) => void;
-    onChange: (text?: string) => void;
-    onSubmit: (text: string) => void;
+    onCancel: () => void;
+    onChange: (nextEditorState: EditorState) => void;
```

Comment:
Can we pass just the text string up to avoid exposing the internals of the component?